x-pack/platform/plugins/shared/lens/public/shared_components/flyout_container.tsx

```diff
@@ -116,8 +116,7 @@ export function FlyoutContainer({
             if (isOpen) {
               // EuiFocusTrap interferes with animating elements with absolute position:
               // running this onAnimationEnd, otherwise the flyout pushes content when animating.
-              // The EuiFocusTrap is disabled when inline editing as it causes bugs with comboboxes
-              setFocusTrapIsEnabled(!Boolean(isInlineEditing));
+              setFocusTrapIsEnabled(true);
```

Comment:
@stratoula, do you remember exactly what issues we had with the comboboxes?

Reply:
Hmmm no, I need to check Alex